### PR TITLE
Display multiple upcoming events

### PR DESCRIPTION
**Pull request by mikefowler**
Adds the ability to display multiple upcoming events on the homepage. In our
case, and as implemented by these changes, we will display 2 upcoming events.
The static `Event.upcoming` method now accepts a parameter, `count`, to support
returning multiple upcoming events. `count` defaults to 1. Adds or updates
appropriate tests, and slightly changes view logic in the
`home/index.html.erb` view.

### CODE REVIEW

**Comment by mikefowler** [comment]
Attn @jonmagic, if you're around today.

File: app/models/event.rb [hunk 1]
```diff
@@ -71,11 +71,18 @@ def subscribers_to_notify
...
-  # Public: Upcoming event if one exists.
+  # Public: Upcoming events, if they exist
   #
-  # Returns an Event or NilClass.
-  def self.upcoming
-    where("start_time > ?", Time.now).order("start_time ASC").first
+  # count - the number of upcoming events to return
+  #
+  # Returns an array of up to `count` Events or NilClass.
+  def self.upcoming (count = 1)
```
Comment (jonmagic) [inline app/models/event.rb]: `def self.upcoming(count = 1)`, just remove the space

File: app/models/event.rb [hunk 2]
```diff
@@ -71,11 +71,18 @@ def subscribers_to_notify
...
+    events = where("start_time > ?", Time.now).order("start_time ASC")
+    if events.empty?
+      nil
+    else
+      events.take(count)
+    end
   end
```
Comment (jonmagic) [inline app/models/event.rb:81]: I think this logic could be shortened to:

``` ruby
where("start_time > ?", Time.now).order("start_time ASC").first(count)
```

Might as well return an empty array if no upcoming events exist.

File: app/views/home/index.html.erb [hunk 3]
```diff
@@ -28,10 +28,12 @@
   </div>
 </section>
 
-<% if @event.present? %>
+<% if @upcoming_events.present? %>
```
Comment (jonmagic) [inline app/views/home/index.html.erb]: This will return false for an empty array, but for reading purposes I'd probably change `.present?` to `.any?`.

**Comment by jonmagic** [comment]
A few small notes, looking great! Also :heart: the newsletter that went out today, looked great and the content was great.

File: app/models/event.rb (hunk 2 above)
Comment (mikefowler) [inline app/models/event.rb:81]: Figured there was a better way to do this.

**Comment by mikefowler** [comment]
Made suggested edits, @jonmagic. Press the magic button for me when verified.

**Comment by mikefowler** [comment]
Additional note: it seems like tests related to newsletter are failing, at least on my environment. Something off with my ENV setup or what?

**Pull request merged**